packages/swap/src/routes/v2/utils.ts

```diff
@@ -95,7 +94,7 @@ export const getLatestSwapForId = async (id: string) => {
       where: { nativeId: BigInt(id) },
       include: swapRequestInclude,
     });
-  } else if (txHashRegex.test(id)) {
+  } else {
     swapRequest = await prisma.swapRequest.findFirst({
       where: { depositTransactionRef: id },
```

Comment:
i think it would be nice to make this case insensitive

Reply:
totally, i saw a lot of inconsistency when copying hashes from different places (sometimes checksummed,  sometimes not)